pysuccinct/pyquery_file.cpp

```diff
@@ -0,0 +1,98 @@
+#include <cstdio>
+#include <iostream>
+#include <unistd.h>
+#include <ctime>
+#include <sstream>
+#include <sys/time.h>
+
+#include "succinct_file.h"
+
+#include <boost/python.hpp>
+using namespace boost::python;
+
+/**
```

Comment:
print_usage() and print_valid_cmds() are unused/unnecessary and should be removed...

Reply:
Same comments for Shard, Semistructured..